src/program.rs

```diff
@@ -126,6 +126,12 @@ impl Program {
         report(&mut self.cache, error)
     }
 
+    /// Wrapper for [`output_doc`]
```

Comment:
Would that reference work? I suspect it may refer to the very same function. Maybe you'll need `[doc::output_doc]` instead?

Reply:
Both won't work since `doc::output_doc` is in a private module. I simply removed the reference and changed the documentation.